hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ConfigurationService.java

```diff
@@ -357,4 +358,19 @@ default UUID updateAsync() {
      * @return registered FlakeIdGenerator configurations
      */
     Map<String, FlakeIdGeneratorConfig> getFlakeIdGeneratorConfigs();
+
+    /**
+     * Finds existing external DataStore config.
+     *
+     * @param name name of the config
+     * @return External DataStore config or {@code null} when requested configuration does not exist
+     */
+    ExternalDataStoreConfig findExternalDataStoreConfig(String name);
+
+    /**
+     * Returns all registered external DataStore configurations.
```

Comment:
as it's a map, I'd add "keyed by store's name"; easier to reason about the result

Reply:
fixed: https://github.com/hazelcast/hazelcast/pull/21930/commits/3646557931948f04b7a78e8d19829db9873473e0